validator/line/line.go

```diff
@@ -0,0 +1,147 @@
+package line
+
+import (
+    "bufio"
+    "encoding/json"
+    "fmt"
+    "net/url"
+    "regexp"
+    "strconv"
+    "strings"
+
+    "github.com/nextdotid/proof_server/config"
+    "github.com/nextdotid/proof_server/types"
+    "github.com/nextdotid/proof_server/util"
+    mycrypto "github.com/nextdotid/proof_server/util/crypto"
+    "github.com/sirupsen/logrus"
+    "golang.org/x/xerrors"
+
+    "github.com/gotd/td/line"
+    "github.com/gotd/td/ln"
```

Comment:
`go: module github.com/gotd/td@upgrade found (v0.75.0), but does not contain package github.com/gotd/td/ln`